### PR TITLE
W-12771843-vm-bare-metal-part2-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/setup-vmbaremetal-cluster.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
+== Setting your VM/Bare Metal Cluster for Runtime Fabric Installation
+
+MuleSoft received customer feedback asking for rapid patching of high severity vulnerabilities in Kubernetes (K8s) and for MuleSoft to better leverage their existing K8s investments and operational practices. In response to this demand, MuleSoft partnered with industry-leading managed K8s platforms vendors such as AWS, Microsoft, Google and IBM. As a result, MuleSoft is proactively transitioning to deliver K8s to their customers via secure, highly available, and managed K8s platform from the above vendors and will stop investing in the black box approach of Runtime Fabric appliance in which you have found limited visibility and extensibility.
+
+Runtime Fabric is compatible with AWS, Google, Microsoft, and IBM RedHat Openshift solutions that deliver faster and consistent security patches. These integrations enable you to easily leverage existing investments in the Kubernetes ecosystem.
+
+Depending on your affinity to public clouds and in-house Kubernetes expertise, the following self-managed Red Hat OpenShift offerings can enable both On-Prem and public cloud bare-metal solutions:
```
Comment (dhirsch1) [inline modules/ROOT/pages/setup-vmbaremetal-cluster.adoc]:
```suggestion
Depending on your affinity to public clouds and in-house Kubernetes expertise, the following self-managed Red Hat OpenShift offerings can enable both on-premises and public cloud bare-metal solutions:
```

File: modules/ROOT/pages/setup-vmbaremetal-cluster.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+* https://www.redhat.com/en/technologies/cloud-computing/openshift/kubernetes-engine[Red Hat OpenShift Kubernetes engine (Entry level)^]
+* https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform[Red Hat OpenShift Container Platform (OCP)^]
+* https://www.redhat.com/en/technologies/cloud-computing/openshift/platform-plus[Red Hat OpenShift Platform Plus^]
+
+After K8s is setup on the cluster using the solutions previously mentioned, proceed to the Runtime Fabric installation section to setup Runtime Fabric as you would install on an AKS or EKS.
```
Comment (dhirsch1) [inline modules/ROOT/pages/setup-vmbaremetal-cluster.adoc]:
```suggestion
After you use one of these solutions to set up K8s on the cluster, proceed to the Runtime Fabric installation section to set up Runtime Fabric as you would install on an AKS or EKS. 
```

Comment (dhirsch1) [reply modules/ROOT/pages/setup-vmbaremetal-cluster.adoc]: Reworded the above paragraph to not use passive tense.

File: modules/ROOT/pages/setup-vmbaremetal-cluster.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+* https://www.redhat.com/en/technologies/cloud-computing/openshift/kubernetes-engine[Red Hat OpenShift Kubernetes engine (Entry level)^]
+* https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform[Red Hat OpenShift Container Platform (OCP)^]
+* https://www.redhat.com/en/technologies/cloud-computing/openshift/platform-plus[Red Hat OpenShift Platform Plus^]
+
+After K8s is setup on the cluster using the solutions previously mentioned, proceed to the Runtime Fabric installation section to setup Runtime Fabric as you would install on an AKS or EKS. 
+
+
+== What does this change mean for appliance users? 
+Existing appliance utility users: The licensed Core capacity which is currently allocated to a Runtime Fabric appliance deployment can be migrated to any customer-hosted deployment option such as Runtime Fabric on public clouds, OpenShift, or Mule runtime standalone, subject to the total license limitations. Customers with Flexible Hybrid Deployment rights may also convert Cores to vCores to utilize this capacity on CloudHub 2.0.
```
Comment (dhirsch1) [inline modules/ROOT/pages/setup-vmbaremetal-cluster.adoc]:
```suggestion
Existing appliance utility users can migrate the licensed Core capacity currently allocated to a Runtime Fabric appliance deployment to any customer-hosted deployment option such as Runtime Fabric on public clouds, OpenShift, or Mule runtime standalone, subject to the total license limitations. Customers with Flexible Hybrid Deployment rights can also convert Cores to vCores to use this capacity on CloudHub 2.0.
```

File: modules/ROOT/pages/setup-vmbaremetal-cluster.adoc [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+== Available Migration Options
+
+If you are looking for an Integration Platform as a Service, https://www.mulesoft.com/platform/saas/cloudhub-ipaas-cloud-based-integration[Cloudhub 2.0^] is the recommended offering. This Mulesoft flagship offering has the lowest operational overhead.
+
+If you already have knowledge in public clouds and have some experience in managing a Kubernetes cluster, migrate to xref:1.13@runtime-fabric::index-self-managed.adoc[Runtime Fabric on public cloud Kubernetes clusters]. Runtime Fabric is currently supported on the following public clouds:
```
Comment (dhirsch1) [inline modules/ROOT/pages/setup-vmbaremetal-cluster.adoc]:
```suggestion
If you already have knowledge about public clouds and have some experience in managing a Kubernetes cluster, migrate to xref:1.13@runtime-fabric::index-self-managed.adoc[Runtime Fabric on public cloud Kubernetes clusters]. The following clouds support Runtime Fabric:
```

File: modules/ROOT/pages/setup-vmbaremetal-cluster.adoc [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+* https://developer.mulesoft.com/tutorials-and-howtos/runtime-fabric/runtime-fabric-aws-elastic-kubernetes-service/Runtime[Fabric on AWS Elastics Kubernetes Service^]
+* https://developer.mulesoft.com/tutorials-and-howtos/runtime-fabric/runtime-fabric-azure-kubernetes-service/[Runtime Fabric on Google Kubernetes Engine^]
+* https://developer.mulesoft.com/tutorials-and-howtos/runtime-fabric/runtime-fabric-google-kubernetes-engine/[Runtime Fabric on Microsoft Azure Kubernetes Service^]
+
+If you prefer to standardize your own bare-metal solutions using public Kubernetes distributions, follow https://www.youtube.com/watch?v=MYOeX5qjYew[Runtime Fabric supported IBM Openshift offerings^]. These seven offerings include pre-built and certified assets, connectors, and operators, enabling both on-premises and public cloud bare-metal solutions.
```
Comment (dhirsch1) [inline modules/ROOT/pages/setup-vmbaremetal-cluster.adoc]:
```suggestion
To standardize your own bare-metal solutions using public Kubernetes distributions, follow https://www.youtube.com/watch?v=MYOeX5qjYew[Runtime Fabric supported IBM Openshift offerings^]. These offerings include pre-built and certified assets, connectors, and operators, which enable both on-premises and public cloud bare-metal solutions.
```

File: modules/ROOT/pages/setup-vmbaremetal-cluster.adoc [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+** https://azure.microsoft.com/en-us/products/openshift/[Azure Red Hat Openshift (ARO)^]
+** https://cloud.google.com/blog/products/gcp/red-hats-openshift-dedicated-now-generally-available-on-google-cloud[Red Hat OpenShift Dedicated on GCP^]
+** https://www.ibm.com/cloud/openshift[Red Hat OpenShift on IBM Cloud^]
+
+* Self-managed services
+** https://www.redhat.com/en/technologies/cloud-computing/openshift/kubernetes-engine[RedHat OpenShift Kubernetes engine (Entry level)^]
+** https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform[RedHat OpenShift Container Platform (OCP)^]
+** https://www.redhat.com/en/technologies/cloud-computing/openshift/platform-plus[RedHat OpenShift Platform Plus^]
+
+If you don’t have investments in Kubernetes and need to stay in your own-hosted environment, https://www.google.com/url?q=https://www.mulesoft.com/lp/dl/mule-esb-enterprise&sa=D&source=docs&ust=1680125428223709&usg=AOvVaw3DzPT38SO_tIf8PhG-wDtd[Mule runtime standalone^] is an available option. Note that this deployment model has reduced runtime resiliency and heavy management overhead.
```
Comment (dhirsch1) [inline modules/ROOT/pages/setup-vmbaremetal-cluster.adoc]:
```suggestion
If you don’t have investments in Kubernetes and must stay in your own hosted environment, https://www.google.com/url?q=https://www.mulesoft.com/lp/dl/mule-esb-enterprise&sa=D&source=docs&ust=1680125428223709&usg=AOvVaw3DzPT38SO_tIf8PhG-wDtd[Mule runtime standalone^] is an available option. This deployment model has reduced runtime resiliency and heavy management overhead.
```

Comment (dhirsch1) [reply modules/ROOT/pages/setup-vmbaremetal-cluster.adoc]: What does "investments in Kubernetes" mean? If you haven't used Kubernetes?

File: modules/ROOT/pages/setup-vmbaremetal-cluster.adoc [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+== What action can appliance users take?
```
Comment (dhirsch1) [inline modules/ROOT/pages/setup-vmbaremetal-cluster.adoc]: We don't put questions in headers.

**Review by fermujica — APPROVED** [review]

**Pull request merged**